bios/dan.md

```diff
@@ -1,5 +1,5 @@
 # Dan Clayton
-![Picture of Dan](https://www.facebook.com/photo.php?fbid=10155443341465333&l=54857258d0)
+![Picture of Dan](https://www.facebook.com/plugins/post.php?href=https%3A%2F%2Fwww.facebook.com%2Fphoto.php%3Ffbid%3D10155443341465333%26set%3Da.10150264811455333%26type%3D3&width=500" width="500" height="502" style="border:none;overflow:hidden" scrolling="no" frameborder="0" allowTransparency="true" allow="encrypted-media")
```

Comment:
hmmmm i think this is more information than we need and this probably still won't work... i'm coming over!